src/include/btmem.h

```diff
@@ -225,6 +225,7 @@ struct __wt_ovfl_reuse {
 #define WT_HS_CONFIG                                                   \
     "key_format=" WT_HS_KEY_FORMAT ",value_format=" WT_HS_VALUE_FORMAT \
     ",block_compressor=" WT_HS_COMPRESSOR                              \
+    ",log=(enabled=false)"                                             \
```

Comment:
Should we add a comment here?

Reply:
Added a comment in cfbffdf046b920ca909deb3db9dc7a9a30d717f5